README.md

```diff
@@ -1,60 +1,67 @@
 <div align="center">
     <div>
         <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
-        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="200px">
+        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
     </div>
 </div>
 
 # NI JavaScript and TypeScript Style Guide
+
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config.svg)](https://www.npmjs.com/package/@ni/eslint-config)
+
 Welcome to NI's JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started).
 
 ## Installation
 
 Install `@ni/eslint-config` and its peer dependencies.
 
-Use [npm view](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
+Use [`npm view`](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
 
 ```bash
 npm view @ni/eslint-config peerDependencies
 ```
 
-Alternatively, use [install-peerdeps](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
+Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
 
 ```bash
 npx install-peerdeps --dev @ni/eslint-config
 ```
 
-### Angular
+### Additional Angular-specific Installation
```

Comment:
There's no need for "-specific". "Angular Installation" is fine and simple if "Installation" is needed at all. I prefer less words in the instructions section of a README, because most users don't care and only want exactly what they need to do and fast.

Reply:
If you look how Github renders it on https://github.com/ni/javascript-styleguide you can see that clicking the first title called angular gives the url with #angular and the second title called angular gives the anchor #angular-1. 

I like to avoid having duplicate titles to avoid usage of the indexed anchors / the need of custom anchors. I went with the option for 'Angular Installation' but if you feel strongly about keeping the indexed anchor + a custom anchor I don't feel super strongly about switching back.

Reply:
I don't thinks it's a big enough issue in this case. However, the preferred UX should take precedence over such small additional effort unless there's some more serious reason for avoiding indexed anchors.